Github-Issue-Guide.md

```diff
@@ -0,0 +1,55 @@
+# Filing an Issue on Oppia's GitHub: A Step-by-Step Guide for Non-Developers
+
+Welcome to Oppia's GitHub issue filing guide for non-developers! This step-by-step guide will help you navigate the process of creating a GitHub account and filing an issue on Oppia's GitHub repository. By following these instructions, you can contribute to improving Oppia without worrying about making mistakes. Let's get started!
+
+## Create a GitHub Account
+
+1. Visit the GitHub website at [https://github.com](https://github.com).
+
+2. Click on the "Sign up" button to create a new GitHub account.
+
+3. Follow the instructions provided by GitHub to complete the signup process.
+
+4. Verify your account by clicking on the confirmation link sent to your email.
+
+## Access Oppia's GitHub Repository
+
+1. Open your web browser and go to [Oppia's GitHub repository](https://github.com/oppia/oppia).
+
+2. You will see the main page of the Oppia repository where the development and issue tracking take place.
+
+## File a New Issue
+
+1. On the Oppia repository page, locate the "Issues" tab near the top navigation bar and click on it.
+ <img width="1437" alt="Screenshot 2023-06-08 at 23 50 59" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/1e0e3833-d04b-4507-8251-f6039a094329">
+
+2. In the "Issues" tab, you will find a green button labeled "New issue." Click on it to create a new issue.
+<img width="1440" alt="Screenshot 2023-06-08 at 23 51 58" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/05b279f5-24b9-4986-a7a9-352e882198f1">
+
+3. Choose type of the issue which you want to file from the list of available types.
+<img width="1440" alt="Screenshot 2023-06-08 at 23 53 15" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/e4ac8fd7-858f-41ca-84c1-0b7a32f425d6">
+
+4. On the issue creation page, provide a concise and descriptive title for your issue in the "Title" field and follow the steps as mentioned along with each comment box.
+<img width="1440" alt="Screenshot 2023-06-09 at 00 00 18" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/0a578025-cef5-4a6b-bb7b-dbf0ba0b8887">
+
+5. If applicable, you can include screenshots or any other supporting files by dragging and dropping them into the comment box.
+
+6. Preview your issue by clicking on the "Preview" tab and make any necessary edits.
```

Comment:
There is no "preview" option for issue forms (Bug reports and Feature requests). So we can remove this step.